pytensor/tensor/rewriting/basic.py

```diff
@@ -975,6 +976,30 @@ def local_join_make_vector(fgraph, node):
         return [ret]
 
 
+@register_specialize
+@register_canonicalize
+@register_useless
+@node_rewriter([Sum])
+def local_sum_make_vector(fgraph, node):
+    """A sum of a MakeVector node is just the sum of the elements."""
+    (array,) = node.inputs
+
+    if array.owner is None:
+        return
+
+    if not isinstance(array.owner.op, MakeVector):
+        return
+
+    if node.op.axis not in [None, 0, -1]:
+        return
+
+    elements = array.owner.inputs
+    dtype = node.op.acc_dtype
+    element_sum = add(*[cast(value, dtype) for value in elements])
+
+    return [as_tensor_variable(element_sum)]
```

Comment:
`as_tensor_variable` is redundant, `add` must return one

Reply:
ups, leftover from when I didn't do it in an Elemwise...